.github/workflows/rate-limiting.yml

```diff
@@ -52,12 +60,18 @@ jobs:
         run: |
           echo "tsa_staging=success" >> $GITHUB_OUTPUT
           rate-limiting --url https://timestamp.sigstage.dev/api/v1/timestamp/certchain || echo "tsa_staging=failed" >> $GITHUB_OUTPUT
-      - name: Rate Limit Rekor Prod
-        id: rekor_prod
+      - name: Rate Limit Rekor v1 Prod
+        id: rekor_v1_prod
+        continue-on-error: true
+        run: |
+          echo "rekor_v1_prod=success" >> $GITHUB_OUTPUT
+          rate-limiting --url https://rekor.sigstore.dev/api/v1/log || echo "rekor_v1_prod=failed" >> $GITHUB_OUTPUT
+      - name: Rate Limit Rekor v2 Prod
+        id: rekor_v2_prod
         continue-on-error: true
         run: |
-          echo "rekor_prod=success" >> $GITHUB_OUTPUT
-          rate-limiting --url https://rekor.sigstore.dev/api/v1/log || echo "rekor_prod=failed" >> $GITHUB_OUTPUT
+          echo "rekor_v2_prod=success" >> $GITHUB_OUTPUT
+          rate-limiting --url https://log2025-1.rekor.sigstore.dev/healthz || echo "rekor_v2_prod=failed" >> $GITHUB_OUTPUT
```

Comment:
No need to change this for this PR, but we should make this dynamic where we fetch the latest shard from the signing config. Could you file an issue to track this?